apps/sage_intacct/migrations/0008_auto_20210408_0812.py

```
@@ -0,0 +1,25 @@
+# Generated by Django 3.0.3 on 2021-04-08 08:12
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sage_intacct', '0007_auto_20210308_0759'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='chargecardtransaction',
+            name='reference_no',
+            field=models.CharField(default='dummy_reference_no', help_text='Sage Intacct Reference number to transaction', max_length=255),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='chargecardtransaction',
+            name='vendor_id',
+            field=models.CharField(default='dummy_vendor_id', help_text='Sage Intacct Vendor ID', max_length=255),
```

defaulting to a dummy value, fyi